battleship-client/src/main/java/com/epam/solid/nie/client/ui/GameScene.java

```diff
@@ -72,15 +84,10 @@ private void makeEnemyMove(Cell cell1) {
             int x = cell1.getCellX();
             int y = cell1.getCellY();
             Cell cell = playerBoard.getCell(x, y);
-            if (cell.wasShot) {
+            if (cell.wasShot)
```

Comment:
braces

Reply:
why do you consider lack of braces as mistake - in my opinion it is one of matters which should be determine by each team

Reply:
In my opinion (and I've heard/read same many times) they're highly recommended for readability and for lowering the risk of introducing bugs when expanding a simple logic. If you're strongly opposed to it, then you can ignore it..

Reply:
It IS your choice and preference. However, many errors were caused by lack of braces already and adding "just one line" later on. And no - you don't need to change if your team doesn't want to.

Reply:
https://github.com/smolamarcin/battleship/issues/151